### PR TITLE
[WDFT - Jan 22] Day 5 Lab 1 - Doan Tran

### PR DESCRIPTION
**Pull request by Dorrrn**
Iteration 1-3 done

### CODE REVIEW

**Comment by Dorrrn** [comment]
wow, what a syntax challenge :D

File: src/viking.js [hunk 1]
```diff
@@ -1,16 +1,108 @@
...
+// Show Status
+function showStatus() {
+  if (this.saxonArmy.length < 1 && this.vikingArmy.length > 0) {
+    return 'Vikings have won the war of the century!';
+  } else if (this.vikingArmy.length < 1 && this.saxonArmy.length > 0) {
+    return 'Saxons have fought for their lives and survived another day...';
+  } else if (this.saxonArmy.length === 1 && this.vikingArmy.length === 1) {
+    return 'Vikings and Saxons are still in the thick of battle.';
+  }
+}
```
Comment (TA-Remote) [inline src/viking.js:103]: Your `showStatus `method should be inside War class ;)

**Review by TA-Remote — COMMENTED** [review]
Great work Doan! 

In the last iteration, as Luis suggested today you can make code shorter and more readable by separatingthe generic logic. For example inside War class you can create   `pickRandom(army) {}` method . And call that method instead line 62, 65 (by passing the needed army) etc.

File: src/viking.js (hunk 1 above)
Comment (Dorrrn) [reply src/viking.js:103]: Ahhh !!!  that makes sense now :D

Comment (Dorrrn) [reply src/viking.js:103]: As always, thanks a lot for reviewing !! :)

**Pull request closed without merging**